torch/csrc/jit/operator_upgraders/README.md

```diff
@@ -0,0 +1,95 @@
+# Guidance for Operator Developer
```

Comment:
I think Tugsuu is also adding this file in his PR:
https://github.com/pytorch/pytorch/pull/72313/files

Could you coordinate with him to land it? Thanks

Reply:
Yeah @tugsbayasgalan  reached out to me and mentioned about it. Sorry should've checked with him so there isn't overlap. I will address the comments from you and update this pr.

Reply:
https://github.com/pytorch/pytorch/pull/72313 includes code changes to skip bc check for operators with valid upgraders, as well as the guidance. It's probably better to have one dedicated pr for the guidance. I've added the changes from that pr to here and will address the feedback too.

Reply:
Yeah. I was updating our FC/BC checking test and I needed to reference this doc so I included my own version in my PR. But I think it makes sense to land this PR first